ci/release/update-version.sh

```diff
@@ -87,6 +87,7 @@ DEPENDENCIES=(
 for DEP in "${DEPENDENCIES[@]}"; do
   for FILE in dependencies.yaml conda/environments/*.yaml; do
     sed_runner "/-.* ${DEP}==/ s/==.*/==${NEXT_SHORT_TAG_PEP440}.*/g" ${FILE}
+    sed_runner "/-.* ${DEP}-cu.*==/ s/==.*/==${NEXT_SHORT_TAG_PEP440}.*/g" ${FILE}
```

Comment:
Can we tighten this regex? I don't want this to catch `cu` prefixes like `cudf`, only `cu11` / `cu12` and so forth.
```suggestion
    sed_runner "/-.* ${DEP}-cu[0-9][0-9]==/ s/==.*/==${NEXT_SHORT_TAG_PEP440}.*/g" ${FILE}
```

Reply:
I'd like to align on a strategy here that we can copy across all of RAPIDS. If we think there's a better way to optionally catch `-cu11` without adding a second `sed` command, I'd be open to that. Perhaps this can use some kind of optional `-cu[0-9][0-9]` group that can match nothing, or a `-cu11` / `-cu12` suffix.